pike/core.py

```diff
@@ -413,7 +447,7 @@ def __str__(self):
 
     def _value_str(self, value):
         if isinstance(value, array.array) and value.typecode == 'B':
-            return '0x' + ''.join(map(lambda b:'%.2x'%b,value))
+            return '0x' + ''.join(['%.2x'%b for b in value])
```

Comment:
I think a generator expression would be fine here (i.e. omitting the `[]`):
``` python
>>> '.'.join([str(i) for i in range(4)])
'0.1.2.3'
>>> '.'.join(str(i) for i in range(4))
'0.1.2.3'
```

Reply:
fixed